addons/event_product/models/event_type_ticket.py

```diff
@@ -22,11 +23,27 @@ def _default_product_id(self):
         domain=[("service_tracking", "=", "event")], default=_default_product_id)
     currency_id = fields.Many2one(related="product_id.currency_id", string="Currency")
     price = fields.Float(
-        string='Price', compute='_compute_price',
+        string='Ticket Price', compute='_compute_price',
         digits='Product Price', readonly=False, store=True)
     price_reduce = fields.Float(
         string="Price Reduce", compute="_compute_price_reduce",
         compute_sudo=True, digits='Product Price')
+    additional_product_ids = fields.Many2many('product.product', string='Additional Products',
+            domain=[('service_tracking', '!=', 'event')],
```

Comment:
Strangely, the domain doesn't seem to work 🤔 

<img width="550" height="324" alt="Image" src="https://github.com/user-attachments/assets/e543a9c8-dede-4142-805d-68523bc9062c" />

Reply:
The goal was to remove the product linked to event ticket which seems to work here 🤔